runconfig/parse.go

```diff
@@ -110,6 +112,8 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 
 	cmd.Var(&flAttach, []string{"a", "-attach"}, "Attach to STDIN, STDOUT or STDERR")
 	cmd.Var(&flBlkioWeightDevice, []string{"-blkio-weight-device"}, "Block IO weight (relative device weight)")
+	cmd.Var(&flReadBpsDevice, []string{"-read-bps-device"}, "Specifies upper limit on READ rate from the device")
```

Comment:
[]string{"--read-bps-device="<device-path>:<limit>[unit]" } , "Limits the read rate from a device.  Limit is a positive integer. Unit can be one of kb, mb, or gb."

[]string{"--write-bps-device="<device-path>:<limit>[unit]" } , "Limits the write rate to a device.  Limit is a positive integer. Unit can be one of kb, mb, or gb."